src/tests/test_config.py

```diff
@@ -93,5 +94,6 @@ def test_reset(self):
             BMConfigParser().safeGetInt('bitmessagesettings', 'maxaddrperstreamsend'), 100)
```

Comment:
split into 2 tests.

one, tests `_reset`, after reset, the `settingsversion` section must be absent.

second, let's call it `test_defaults`
```
config = BMConfigParser()
config.addsection("bitmessagesettings")
config.set("bitmessagesettings", "maxaddrperstreamsend", 100)
config.read() # this resets plus populates with defaults
assert(config.get("bitmessagesettings", "maxaddrperstreamsend"), 500)
```